gprofiler/main.py

```diff
@@ -834,6 +837,17 @@ def parse_cmd_args() -> configargparse.Namespace:
         "The file modification indicates the last snapshot time.",
     )
 
+    if is_linux() and not is_aarch64():
```

Comment:
We also need the detection for AMD too. I explored the platform module in python but none of the existing functions provide the distinction btw AMD and Intel architecture. I think we should parse "/proc/cpuinfo".

Reply:
Actually, I found a better module to detect it. It's py-cpuinfo https://pypi.org/project/py-cpuinfo/
cpuinfo.get_cpu_info()['vendor_id_raw'] gives the info we need.
>>> cpuinfo.get_cpu_info()['vendor_id_raw']
'GenuineIntel'